plugins/task/src/index.ts

```diff
@@ -31,11 +31,18 @@ export const TaskStatuses = {
 export type TaskStatus = typeof TaskStatuses[keyof typeof TaskStatuses]
 
 export interface Task extends Doc {
-  shortRefId: Ref<ShortRef>
-  name: string
-  description: string
-  assignee?: Ref<Account>
-  status: TaskStatus
+  shortRefId: Ref<ShortRef>,
+  name: string,
+  description: string,
+  assignee?: Ref<Account>,
+  status: TaskStatus,
+  subtasks: Array<Subtask>,
+  commentSpace: Ref<Space>
+}
+
+export interface Subtask {
```

Comment:
I'm not like Subtask name, since it is not a subtask.

Let's make it `CheckListItem` and put them inside object itself. I've not think we need them outside of Task.